app/src/main/java/ch/epfl/sdp/blindwar/data/music/metadata/MusicMetadataRepository.kt

```diff
@@ -101,4 +79,23 @@ class MusicMetadataRepository {
             )
         }
     }
+
+    /** Put these functions in string util **/
+    private fun capitalizeString(string: String): String {
+        return string.split(" ").joinToString(" ") { w -> w.lowercase().replaceFirstChar {c -> c.uppercase()}}

```

Comment:
maybe use .capitalize() instead of .replaceFirstChar{ c -> c.uppercase()} would be simpler

Reply:
capitalize() is apparently deprecated in the Kotlin version we are using